config/samples/numaplane.numaproj.io_v1alpha1_monovertexrollout.yaml

```diff
@@ -2,19 +2,21 @@ apiVersion: numaplane.numaproj.io/v1alpha1
 kind: MonoVertexRollout
 metadata:
   name: my-monovertex
```

Comment:
did you mean to remove namespace? if you did, can you remove it from all 4 example Rollouts?

Reply:
Those were for my testing, accidentally committed, will remove.